fetch_and_process_api_call/Makefile

```diff
@@ -0,0 +1,16 @@
+default: run
+
+tmp:
+	@mkdir -p tmp
+	
+tmp/x.wasm: main.go | tmp
+	tinygo build -o tmp/x.wasm -target=wasi main.go
+
+build: tmp/x.wasm
+
+run: build
+	cat fn-call.json | ./bky-as attest-fn-call  | jq -r .function_calls[0].claims.output | base64 -d | jq .
```

Comment:
consider updating so that this doesn't require `bky-as` to be in the root of the directory.  In the quick examples I was creating, that worked, but for this one, it could be a pain.

consider updating so that the base64 decode is part of the first jq call so that you can avoid the call to `base64`.  Generally, on stuff I share, I tend to avoid using the base64 command unless it is needed because it is one of the commands that are different between gnu and bsd.

Reply:
OK, I baked in base64 to the jq command string and added the current directory to the path before invoking `bky-as`